samples/CosmosDB/audit-cosmosdb-autofailover-georeplication/azurepolicy.json

```diff
@@ -0,0 +1,30 @@
+{
+  "properties": {
+    "mode": "all",
+    "displayName": "Audit Automatic Failover for CosmosDB accounts",
+    "description": "This policy audits Automatic Failover for CosmosDB accounts",
+    "policyRule": {
+      "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.DocumentDB/databaseAccounts"
+          },
+          {
+            "field": "Microsoft.DocumentDB/databaseAccounts/enableAutomaticFailover",
+            "equals": "false"
+          },
+          {
+            "field": "Microsoft.DocumentDB/databaseAccounts/enableMultipleWriteLocations",
+            "equals": "false"
+          }
```

Comment:
Hi Camille. Yes the intention is that the policy will audit when both are false. This is based on the following 2 scenarios:
1) If the multiple master is not configured, the automatic failover flag should be set to 1 to enable automatic failover.
2) If multiple master is configured then in case of outages the failover will happen automatically. 
Thus, the automatic failover is enabled when either the automatic failover flag is set to 1 or when multiple master is enabled. If both are false, then the policy should audit.

Reply:
Great! Thanks for the explanation :)